src/test/java/com/virtuslab/archunit/ForbiddenMethodsTestSuite.java

```diff
@@ -32,6 +32,14 @@ public void no_classes_should_call_Collections_nCopies() {
         .check(importedClasses);
   }
 
+  @Test
+  public void no_classes_should_call_File_exists() {
+    noClasses()
+        .should().callMethod(java.io.File.class, "exists")
+        .because("in most cases, the check you want to do is `isFile` rather than `exists` (what if this is a directory?)")
```

Comment:
I literally don't remember a case when I wanted to know whether this is a file OR directory, ever. And this is exactly what `exists` does xD